content/docs/command-reference/move.md

```diff
@@ -1,9 +1,9 @@
 # move
 
 Rename a file or a directory and modify the corresponding
-[DVC-file](/doc/user-guide/dvc-file-format) (see `dvc add`) to reflect the
-change. If the file or directory has the same name as the corresponding
-DVC-file, it also renames it.
+[`.dvc` file](/doc/user-guide/dvc-file-format) (see `dvc add`) to reflect the
+change. If the file or directory has the same name as the corresponding `.dvc`
+file, it also renames it.
```

Comment:
Another Q for probably @efiop: Will `dvc move` support desp/outs in `dvc.yaml`?

Reply:
@jorgeorpinel Nope, it won't for the same reasons.

Reply:
Good to know. Leaving this unresolved for the record.